src/main/java/relations/objects/LinearCombination.java

```diff
@@ -25,6 +26,11 @@ public boolean add(final LinearTerm<FieldT> term) {
         return terms.add(term);
     }
 
+    public boolean add(int index, final LinearTerm<FieldT> term) {
```

Comment:
Is this function used anywhere? It seems dangerous as it would shift all other values by one, which might lead to unintended effects. If this is to prepend the 1 at the beginning, can we call it specifically prepend or just insert the 1 as a first element.

Reply:
Very strange. There was a time when this was absolutely necessary to be used, but now I can't find the spot. Perhaps it was an old example that I have since removed. I will remove it.